style.css

```diff
@@ -0,0 +1,806 @@
+  background-image: url('images/tnw-small.svg');
+  width: 45px;
+  height: 9px;
+}
+
+.i-stackpath {
+  background-image: url('images/stackpath.svg');
+  width: 75px;
+  height: 9px;
+}
+
+.logo-small {
+  display: none;
+  background-image:
+    linear-gradient(
+      to bottom right,
+      rgb(206, 2, 149),
+      rgb(255, 68, 34) 80%
+    );
+  width: 100px;
+  height: 95px;
+}
+
+.logo-small img {
+  width: 64px;
+  height: 79px;
+}
+
+.top-nav,
+.bottom-nav,
+.top-nav ul,
+.bottom-nav ul {
+  display: flex;
+  align-items: center;
+}
+
+.top-nav,
+.bottom-nav {
+  padding: 1rem 2rem;
+  justify-content: space-between;
+}
+
+.top-nav ul:first-of-type li {
+  margin-right: 1rem;
+}
+
+.top-nav ul:last-of-type li {
+  margin-left: 1rem;
+}
+
+.top-nav {
+  background-color: #000;
+  font-size: 1.3rem;
+  color: #999;
+  line-height: 1;
+  height: 39px;
+}
+
+.articles ul {
+  clear: both;
+  padding-top: 20px;
+  padding-left: 16px;
+  list-style-type: circle;
+  line-height: 2;
+  color: black;
+}
+
+.bottom-nav ul:first-of-type {
+  padding: 1.5rem 0 1.5rem 2rem;
+}
+
+.bottom-nav ul:first-of-type li {
+  margin-right: 2rem;
+}
+
+.bottom-nav ul:last-of-type li {
+  display: flex;
+  border-left: 1px solid rgba(153, 153, 153, 0.1);
+}
+
+.top-nav ul:first-of-type li:first-child {
+  color: #fff;
+}
+
+.bottom-nav {
+  padding: 0;
+  text-transform: uppercase;
+  color: #999;
+  background-color: whitesmoke;
+  height: 43px;
+}
+
+.bottom-nav button {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  position: relative;
+}
+
+.social-links .icon {
+  width: 1.8rem;
+  height: 1.8rem;
+}
+
+.bottom-nav li:first-of-type .icon {
+  transform: rotate(90deg);
+  margin-top: 0;
+}
+
+.bottom-nav ul:last-of-type {
+  align-self: stretch;
+  align-items: stretch;
+}
+
+.bottom-nav ul:last-of-type li:last-of-type {
+  display: none;
+}
+
+.yellowBar {
+  font-size: 1.3rem;
+  line-height: 1.3;
+  background-color: rgb(255, 221, 51);
+}
+
+@media (max-width: 1024px) {
+  .yellowBar {
+    padding-left: 10rem;
+  }
+
+  .logo-small {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    grid-row: 1 / span 2;
+    float: left;
+    height: 83px;
+  }
+}
+
+@media (max-width: 575px) {
+  .yellowBar {
+    padding-left: 0;
+    padding-right: 0;
+    text-align: center;
+  }
+}
+
+@media (max-width: 480px) {
+  .yellowBar {
+    font-size: 1rem;
+  }
+}
+
+@media (max-width: 1024px) {
+  .yellowBar {
+    padding-left: 10rem;
+  }
+}
+
+@media (max-width: 575px) {
+  .yellowBar {
+    padding-left: 0;
+    padding-right: 0;
+    text-align: center;
+  }
+}
+
+@media (max-width: 480px) {
+  .yellowBar {
+    font-size: 1rem;
+  }
+}
+
+.corona-info {
+  background-image: url('https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Untitled-design28-796x417.png');
+  grid-row-start: 1;
+  grid-row-end: 3;
+  background-position: center center;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  background-size: 100% 100%;
```

Comment:
- [ ] Don't distort the background images. If you set both the width and the height the image will get distorted. Chose either the width or the height so that it keeps its proportion. That goes for every image that is distorting at resize.
![Screenshot_2020-03-22 The Next Web(1)](https://user-images.githubusercontent.com/16193212/77241635-beb49a80-6bd3-11ea-9a3e-0d22c52bb95b.png)
![Screenshot_2020-03-22 The Next Web(2)](https://user-images.githubusercontent.com/16193212/77241636-bf4d3100-6bd3-11ea-8caa-0365696dc21c.png)